dev/crud.html

```diff
@@ -12,6 +12,7 @@
     <script type="module">
       import '@vaadin/crud';
       import '@vaadin/radio-group';
+      import '@vaadin/vaadin-lumo-styles/lumo.css';
```

Comment:
Please rebase and update to use `<link rel="stylesheet">` like in other dev pages.